### DISCUSSION TITLE
Why XMl and not XML or Xml?

### DISCUSSION THREAD

**Discussion by ThePlenkov** [NaturalIntelligence/fast-xml-parser#693] [Q&A]
Hi! I'm very sorry, honestly do not want to seem a nerd =) But i just noticed that you intentionally using XMl notation which looks like XMI. So just curious why not XML for example?

It doesn't impact anything of course - i'm just trying to understand - was it chosen on purpose?

![image](https://github.com/user-attachments/assets/afb38568-6449-4b54-a1a3-aadcfc4e9d20)
Thanks!

**Comment by amitguptagwl** [comment]
If I understand you correctly, your findings are related to the demo or test examples not parsing logic, syntax, or its behavior? Though, it may be a typo or knowingly done for testing in documents, demo or any other text only section. Let me know if it affects code in any way.

**Reply by ThePlenkov** [reply]
That's correct. I only saw it on the site, it's not about logic. But I saw this notation XMl is used in several places that's how I thought it might mean something for the author. If you want I can propose an MR
